integration_tests/tests/s3-standardised-lambda-tests.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  copyObject,
+  checkIfFileExists,
+  deleteObjectInS3,
+  getS3ItemsList,
+} from "../helpers/s3Helper";
+import { resourcePrefix } from "../helpers/envHelper";
+import { checkGivenStringExistsInLogs } from "../helpers/cloudWatchHelper";
+import { waitForTrue } from "../helpers/commonHelpers";
+
+const testStartTime = new Date().getTime();
+const prefix = resourcePrefix();
+const rawinvoiceBucketName = `${prefix}-raw-invoice-pdf`;
```

Comment:
Trivial request, can you capitalise the 'i' in rawinvoiceBucketName and rawinvoiceBucketKey?